src/webgpu/shader/execution/expression/call/builtin/unpack2x16unorm.spec.ts

```diff
@@ -17,7 +17,7 @@ import {
   unpack2x16unorm,
   vec2,
 } from '../../../../../util/conversion.js';
-import { fullU32Range, quantizeToF32 } from '../../../../../util/math.js';
```

Comment:
Fixing small issues I noticed when duplicating this test as a starting point for unpack4x8unorm.spec.ts